lib/ii.c

```diff
@@ -6202,20 +6207,20 @@ grn_ii_update_one(grn_ctx *ctx, grn_ii *ii, grn_id tid, grn_ii_updspec *u, grn_h
           GRN_TEXT_INIT(&term, 0);
           grn_ii_get_term(ctx, ii, tid, &term);
           MERR("%s failed to allocate a buffer: "
-               "<%.*s>: "
-               "<%.*s>(%u): "
-               "(%u:%u): "
-               "segment:<%u>, "
-               "free:<%u>, "
-               "required:<%u>",
-               tag,
-               name_size, name,
-               (int)GRN_TEXT_LEN(&term), GRN_TEXT_VALUE(&term),
-               tid,
-               u->rid, u->sid,
-               pos,
-               b->header.buffer_free,
-               size);
+              "<%.*s>: "
+              "<%.*s>(%u): "
+              "(%u:%u): "
+              "segment:<%u>, "
+              "free:<%u>, "
+              "required:<%u>",
+              tag,
+              name_size, name,
+              (int)GRN_TEXT_LEN(&term), GRN_TEXT_VALUE(&term),
+              tid,
+              u->rid, u->sid,
+              pos,
+              b->header.buffer_free,
+              size);
```

Comment:
Could you revert needless indentation diffs like this and others?

Reply:
Reverted.